apps/src/studio/skins.js

```diff
@@ -931,8 +931,47 @@ function loadHoc2015(skin, assetUrl) {
     'applause'
   ];
 
+  skin.soundGroups = {
+    'R2-D2sound': {
+      randomValue: 'R2-D2random',
+      minSuffix: 1,
+      maxSuffix: 9
```

Comment:
It is to say that the group is comprised of sounds based on the key name (in this case, `R2-D2sound`) and a numeric suffix that increments between `minSuffix` and `maxSuffix`. We could derive it, but it would involve linear runtime searching against `skin.sounds`. I was also trying to be flexible around various skins that could use 0-based vs. 1-based suffixes.

The best way would be to restructure the whole `skin.sounds` as object (and maybe include metadata at the same time?), but I wasn't trying not be so disruptive.